From 51c8a5a568b2cd1dfcb53ed3b49a0f8899be7be4 Mon Sep 17 00:00:00 2001
From: Alexander Kanavin <alex.kanavin@gmail.com>
Date: Mon, 6 Jan 2020 12:44:42 +0100
Subject: [PATCH] meson.build: use 'python3' directly for python

This avoids a dependency on target python (due to meson probing
its configuration).

Upstream-Status: Inappropriate [oe-core specific]
Signed-off-by: Alexander Kanavin <alex.kanavin@gmail.com>
---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index 193ba32b..c01cae65 100644
--- a/meson.build
+++ b/meson.build
@@ -77,7 +77,7 @@ flags = [
 
 add_project_arguments(cc.get_supported_arguments(flags), language : 'c')
 
-prog_python = import('python').find_installation('python3')
+prog_python = 'python3'
 
 not_found = dependency('', required: false)
 gbm_dep = not_found
